src/frontend/js/modules/file.js

```diff
@@ -5,6 +5,7 @@
 
   .constant('FILES_API_URL', '/api/files')
   .constant('DEFAULT_FILE_TYPE', 'application/octet-stream')
+    .constant('MAX_SIZE_UPLOAD_DEFAULT', 104857600) // is equal to 100 MO
```

Comment:
What is MO?

Reply:
Mega Octet, works both for English and French.